test/services/smart-action-hook.test.js

```diff
@@ -73,21 +62,164 @@ describe('services > smart-action-hook', () => {
+          await expect(response).toStrictEqual(expected);
+          expect(hook).toHaveBeenNthCalledWith(
+            1,
+            {
+              fields,
+              record: {},
+              changedField: null,
+            },
+          );
+        });
+
+        it('should assign null value if no one is provided', async () => {
+          expect.assertions(2);
+
+          const { smartActionHook } = initContext().inject();
+          const field = {
+            field: 'myField',
+            type: 'String',
+            value: 'foo',
+          };
+          const addedField = {
+            field: 'myNewField',
+            type: 'String',
+          };
+
+          const fields = [field];
+          const expected = [field, {
+            ...addedField,
+            value: null,
+          }];
+
+          const hook = jest.fn(() => ([field, addedField]));
+
+          const response = await smartActionHook.getResponse({}, hook, fields, {});
+          await expect(response).toStrictEqual(expected);
+          expect(hook).toHaveBeenNthCalledWith(
+            1,
+            {
+              fields,
+              record: {},
+              changedField: null,
+            },
+          );
+        });
+
+        it('should throw an error if the new field is malformed', async () => {
+          expect.assertions(1);
+
+          const { smartActionHook } = initContext().inject();
+          const action = { name: 'actionTest' };
+
+          const field = {
+            field: 'myField',
+            type: 'String',
+            value: 'foo',
+          };
+          const addedField = {
+            type: 'String',
+          };
+
+          const fields = [field, addedField];
+
+          const hook = jest.fn(() => ([field, addedField]));
+
+          await expect(smartActionHook.getResponse(action, hook, fields, {}))
+            .rejects.toThrow(`field attribute inside fileds array on the smart action "${action.name}" must be defined.`);
+        });
+
+        it('should throw an error if the fiel have an undefined hook', async () => {
+          expect.assertions(1);
+
+          const { smartActionHook } = initContext().inject();
+          const action = { name: 'actionTest', hooks: { change: {} } };
+
+          const field = {
+            field: 'myField',
+            type: 'String',
+            value: 'foo',
+          };
+          const addedField = {
+            type: 'String',
+            field: 'myNewField',
+            hook: 'undefinedHook',
+          };
+
+          const fields = [field, addedField];
+
+          const hook = jest.fn(() => ([field, addedField]));
+
+          await expect(smartActionHook.getResponse(action, hook, fields, {}))
+            .rejects.toThrow(`The hook "${addedField.hook}" of "${addedField.field}" field on the smart action "${action.name}" is not defined.`);
+        });
+      });
+
+      it('should have one less filed', async () => {
```

Comment:
```suggestion
      it('should have one less field', async () => {
```
+ I think you are missing a describe here 🤔